api/main.py

```diff
@@ -655,3 +656,15 @@ async def put_regression(regression_id: str, regression: Regression,
         sub_app.app.add_exception_handler(
             errors.InvalidId, invalid_id_exception_handler
         )
+
+
+@app.middleware("http")
+async def redirect_http_requests(request: Request, call_next):
+    """Redirect request with '/latest' version prefix when no
+    version is provided in the URL"""
```

Comment:
I think this needs to fit on a single line:
https://peps.python.org/pep-0257/#multi-line-docstrings

> The summary line may be used by automatic indexing tools; it is important that it fits on one line and is separated from the rest of the docstring by a blank line.

In fact this is a one-line docstring here so by definition it should fit on one line:
https://peps.python.org/pep-0257/#one-line-docstrings

It's basically the same convention as for Git commits.

Reply:
Okay. I've rephrased it to fit it in a single line.